README.md

```diff
@@ -2,4 +2,6 @@
 This is a fun project where you can choose from three options Victory celebration , Rage between drivers and steward or team principle or other fellow driver or crashes
 on choosing Victory Celebration we will get ine f the most epic victory celebration and most epic results
 on choosing  Rage the rage clips the crashes will be potrayed in a funny way
-If you love F1 this is a one stop for you fun, nostalgia, adrenaline in a go.
\ No newline at end of file
+If you love F1 this is a one stop for you fun, nostalgia, adrenaline in a go.
+                                                                                    -Created by
+                                                                                     Manvesh Thakur
```

Comment:
Consider adding a newline at the end of the file. It's a common convention to end a file with a newline. This is because some utilities have problems processing the last line of a file if it isn't newline terminated. [medium]